app/models/user.rb

```diff
@@ -50,6 +50,12 @@ def self.search(term)
     where('username LIKE ?', "%#{sanitize_sql_like(term)}%")
   end
 
+  # Safely gets the user's trust level even if they don't have a community user
+  # @return [Integer] user's trust level
+  def trust_level
+    community_user&.trust_level || 0
```

Comment:
Had to restore the `trust_level` method on user to avoid having to explicitly check that the community user exists